qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -94,6 +94,7 @@
                 - test_smap:
                     # support RHEL.7 guest or later
                     no RHEL.6
+                    unsupported_models = "Haswell Haswell-noTSX IvyBridge SandyBridge Westmere Nehalem Penryn Conroe Opteron_G4 Opteron_G3 Opteron_G2 Opteron_G1"
```

Comment:
@nanliu-r Shall we split it into two lines?

Reply:
> @nanliu-r Shall we split it into two lines?

@vivianQizhu  I split AMD and Intel into different lines, thanks.